httpx/__init__.py

```diff
@@ -58,6 +59,7 @@
     "DecodingError",
     "HTTPError",
     "InvalidURL",
+    "MissingDependencies",
```

Comment:
I don't think we should introduce exception classes for this sort of case. A `RuntimeError` or something similar would be suitable here.

Reply:
Fair enough :-) Dropped this exception class in the latest commit.